UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs

```diff
@@ -356,7 +356,12 @@ public static SpriteUpdateMessage.NetMessage Deserialize(this NetworkReader read
 						Color TheColour = reader.ReadColor();
 						if (ProcessSection)
 						{
-							SP.SetColor(TheColour, false);
+							if (SP)
+							{
+								//TODO: remove this check - registering arrives after the sprite update, all clients will disconnect after a runtime
```

Comment:
it is because SP = null, 
looks like the problem is the original SP is not unregistered 
so it returns a destroyed instance of it